data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt

```diff
@@ -0,0 +1,41 @@
+4.00000e+01 4.00000e+01 7.77713e-43 1.46595e-42 3.06558e-42 5.69261e-42 8.72405e-42 3.85749e-42 1.10618e-42
+4.30000e+01 4.30000e+01 7.76223e-43 1.52087e-42 3.14899e-42 5.68472e-42 8.75318e-42 4.37499e-42 1.21988e-42
+4.60000e+01 4.60000e+01 7.92537e-43 1.54351e-42 3.22099e-42 5.83617e-42 8.89556e-42 4.95957e-42 1.31638e-42
+5.50000e+01 5.50000e+01 8.44626e-43 1.59953e-42 3.39593e-42 6.23444e-42 9.46304e-42 6.19132e-42 1.66392e-42
+6.50000e+01 6.50000e+01 9.11722e-43 1.76846e-42 3.72303e-42 6.67123e-42 1.01088e-41 7.45862e-42 2.05450e-42
+7.80000e+01 7.80000e+01 9.87110e-43 1.89406e-42 4.13114e-42 7.45073e-42 1.12826e-41 9.45524e-42 2.54726e-42
+9.10000e+01 9.10000e+01 1.07978e-42 2.09148e-42 4.58095e-42 8.18234e-42 1.25033e-41 1.09430e-41 3.01938e-42
+1.10000e+02 1.10000e+02 1.23599e-42 2.54674e-42 5.34117e-42 9.49256e-42 1.45589e-41 1.32730e-41 3.78861e-42
+1.29000e+02 1.29000e+02 1.37977e-42 2.73050e-42 6.09397e-42 1.09110e-41 1.64415e-41 1.59235e-41 4.33413e-42
+1.55000e+02 1.55000e+02 1.67911e-42 3.27367e-42 7.20829e-42 1.26651e-41 1.95906e-41 1.93616e-41 5.28994e-42
+1.82000e+02 1.82000e+02 1.87678e-42 3.72534e-42 7.97790e-42 1.44799e-41 2.21773e-41 2.27560e-41 6.15293e-42
+2.19000e+02 2.19000e+02 2.13653e-42 4.29789e-42 9.53187e-42 1.72304e-41 2.65357e-41 2.70078e-41 7.41518e-42
+2.56000e+02 2.56000e+02 2.63127e-42 5.14376e-42 1.10626e-41 2.00144e-41 3.03967e-41 3.16448e-41 8.62069e-42
+3.08000e+02 3.08000e+02 3.12113e-42 5.88562e-42 1.30174e-41 2.33345e-41 3.63719e-41 3.79593e-41 1.04307e-41
+3.61000e+02 3.61000e+02 3.45258e-42 7.29078e-42 1.53722e-41 2.73464e-41 4.16979e-41 4.41771e-41 1.23479e-41
+4.34000e+02 4.34000e+02 4.15748e-42 8.36131e-42 1.81897e-41 3.28317e-41 5.01781e-41 5.43417e-41 1.45845e-41
+5.08000e+02 5.08000e+02 4.88465e-42 9.69197e-42 2.12576e-41 3.75012e-41 5.85624e-41 6.37803e-41 1.72742e-41
+6.12000e+02 6.12000e+02 5.82253e-42 1.19181e-41 2.49416e-41 4.62117e-41 7.07591e-41 7.70188e-41 2.04137e-41
+7.16000e+02 7.16000e+02 6.54451e-42 1.37376e-41 2.96752e-41 5.34004e-41 8.17936e-41 8.96552e-41 2.46678e-41
+8.62000e+02 8.62000e+02 8.09836e-42 1.66308e-41 3.53124e-41 6.39404e-41 9.76279e-41 1.05332e-40 2.92159e-41
+1.00800e+03 1.00800e+03 9.36865e-42 1.88000e-41 4.04923e-41 7.48321e-41 1.13538e-40 1.25845e-40 3.41986e-41
+1.21400e+03 1.21400e+03 1.11054e-41 2.31007e-41 4.91454e-41 8.91463e-41 1.36836e-40 1.52425e-40 4.13885e-41
+1.42000e+03 1.42000e+03 1.33865e-41 2.70491e-41 5.83012e-41 1.06694e-40 1.62034e-40 1.75903e-40 4.75082e-41
+1.71000e+03 1.71000e+03 1.63064e-41 3.14984e-41 7.00930e-41 1.25114e-40 1.91091e-40 2.08317e-40 5.79407e-41
+2.00000e+03 2.00000e+03 1.83104e-41 3.73688e-41 8.18927e-41 1.47614e-40 2.26286e-40 2.45817e-40 6.84680e-41
+5.00000e+03 5.00000e+03 4.72335e-41 9.25169e-41 2.02970e-40 3.58810e-40 5.60517e-40 6.15656e-40 1.71706e-40
+1.00000e+04 1.00000e+04 9.70915e-41 1.83774e-40 4.04142e-40 7.23237e-40 1.11227e-39 1.25377e-39 3.42043e-40
```

Comment:
Why does this file have two mass columns? (or am I mis-reading it?)

Reply:
The LZ arxiv UL is 4.87e-40 @ 10TeV (the sensi numbers agree though-- was the limit overwritten by the mass?)

Reply:
oh and sorry for getting to this only now :)